### PR TITLE
i18n(es): Update of `endpoints.md` 

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Updated content according to this commit e56cbe99deb4842202369ae91b232308e6e59bda

### CODE REVIEW

File: src/pages/es/core-concepts/endpoints.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ export const get: APIRoute = ({ params, request }) => {
 ```
 
 ## Endpoints del Servidor (Rutas de API)
-Todo lo descrito en la sección de endpoints de archivos estáticos también se puede usar en modo SSR: los archivos pueden exportar una función `get` que recibe un objeto con las propiedades `params` y `request`.
+Todo lo descrito en la sección de endpoints de archivos estáticos también se puede usar en modo SSR: los archivos pueden exportar una función `get` que recibe un [objeto de contexto](/es/reference/api-reference/#endpoint-context) con las propiedades similares a las de `Astro` global.
```
Comment (agustinmulet) [inline src/pages/es/core-concepts/endpoints.md:96]:
```suggestion
Todo lo descrito en la sección de endpoints de archivos estáticos también se puede usar en modo SSR: los archivos pueden exportar una función `get` que recibe un [objeto de contexto](/es/reference/api-reference/#endpoint-context) con propiedades similares a las de `Astro` global.
```

File: src/pages/es/core-concepts/endpoints.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ En los sitios generados estáticamente, tus endpoints personalizados son llamado
 ## Endpoints de Archivos Estáticos
 Para crear un endpoint personalizado, agrega un archivo `.js` o `.ts` al directorio de `/pages`. La extensión `.js` o `.ts` se eliminará durante el proceso de compilación, por lo que el nombre del archivo debe incluir la extensión de los datos que se desea generar. Por ejemplo, `src/pages/data.json.ts` generará el endpoint `/data.json`.
 
-Los endpoints exportan una función `get` (opcionalmente `async`) que recibe un objeto con dos propiedades (`params` y `request`) como su único argumento. Esto retorna un objeto con un `body`, y Astro va a llamarlo al momento de compilación y usar sus contenidos del body para generar un archivo.
+Los endpoints exportan una función `get` (opcionalmente `async`) que recibe un [objeto de contexto](/es/reference/api-reference/#endpoint-context) con similares a las de `Astro` global. Esto retorna un objeto con un `body`, y Astro va a llamarlo al momento de compilación y usar sus contenidos del body para generar un archivo.
```
Comment (agustinmulet) [inline src/pages/es/core-concepts/endpoints.md:14]:
```suggestion
Los endpoints exportan una función `get` (opcionalmente `async`) que recibe un [objeto de contexto](/es/reference/api-reference/#endpoint-context) con propiedades similares a las de `Astro` global. Esto retorna un objeto con un `body`, y Astro va a llamarlo al momento de compilación y usar sus contenidos del body para generar un archivo.
```

**Review by agustinmulet — COMMENTED** [review]
Revisado! Cambié los links para el título propuesto por mi en este PR: https://github.com/withastro/docs/pull/1804, decime si te parece bien 😄

**Comment by agustinmulet** [comment]
También faltaría actualizar la indentación que se cambió en este commit https://github.com/withastro/docs/commit/6b72121bde88ce334380592b4a55ca1cc142aacb#diff-c4119b03e79e36340f6ce75858167fc04f8c2efe090c29bd800953964a08bb9d

File: src/pages/es/core-concepts/endpoints.md (hunk 2 above)
Comment (dreyfus92) [reply src/pages/es/core-concepts/endpoints.md]: Muchas gracias por tus comentarios @agustinmulet 😄

**Review by agustinmulet — APPROVED** [review]
LGTM! Although we need to merge this PR first to fix the link checker https://github.com/withastro/docs/pull/1804

**Comment by agustinmulet** [comment]
@dreyfus92 Please keep in mind that there were two commits that updated this file recently. Is it ok for you to make those changes so we keep everything up to date?
The commits are:
- https://github.com/withastro/docs/commit/ce42213f9c9520ffe0081ddf3ceac1c066090206
- https://github.com/withastro/docs/commit/6174e1d820b14af544b5ee1701e9add61cb0c0b4

There are small changes 😄

**Comment by dreyfus92** [comment]
Sure, will handle it in a few.

**Comment by agustinmulet** [comment]
Lovely, LGTM!

**Review by delucis — APPROVED** [review]
Thanks @dreyfus92 and thanks @agustinmulet for keeping track of the updates!

**Pull request merged**